packages/itwinui-react/src/utils/providers/FutureFlagsProvider.tsx

```diff
@@ -16,6 +16,15 @@ export type FutureOptions = {
    * **NOTE**: Since this is a theme bridge to *future* versions, the theme could have breaking changes.
    */
   themeBridge?: boolean;
+  ToggleSwitch?: {
+    /**
+     * By default, `className` and `style` props are applied on the wrapper while other props are applied on the input.
+     * This inconsistency can lead to confusion.
+     *
+     * Thus, setting this flag to `true` or passing `wrapperProps` will apply all props on the input.
+     */
```

Comment:
Suggestion to reword this and frontload the description of the flag.

```suggestion
    /**
     * When `true`, `className` and `style` will be applied on the
     * input element, along with the rest of the DOM props.
     * 
     * By default (without this flag), `className` and `style` get
     * applied on the wrapper element, unless `wrapperProps` is also passed.
     */
```